### PR TITLE
Add update verbs

### PR DESCRIPTION
**Pull request by baldwinn860**
Adds ability to update some entities within robot. Specifically Packages and Subjects. This makes it possible to fix mistakes during initial uploads, like the subject's API or the package's description. Also lets us improve the data of these entities, such as better parentage of packages, and less than useful trace times for subjects.

### CODE REVIEW

File: cmd/robot/build.go [hunk 1]
```diff
@@ -412,3 +418,42 @@ func (v *trackUpdateVerb) Run(ctx context.Context, flags flag.FlagSet) error {
...
+		b := build.NewRemote(ctx, conn)
+		args := flags.Args()
+		pkg := &build.Package{
+			Information: &build.Information{Description: v.Description},
+			Parent:      v.Parent,
+		}
+		if len(args) == 0 {
+			return log.Err(ctx, nil, "Missing argument, must specify a package to update")
+		}
+		err := b.SearchPackages(ctx, script.MustParse("Id == $").Using("$")(args[0]).Query(), func(ctx context.Context, entry *build.Package) error {
```
Comment (pmuetschard) [inline cmd/robot/build.go]: you can combine this into:
```
if err := ...; err != nil {
  ...
}
```

File: cmd/robot/subject.go [hunk 2]
```diff
@@ -103,3 +109,46 @@ func (v *subjectSearchVerb) Run(ctx context.Context, flags flag.FlagSet) error {
...
+		subj := &subject.Subject{
+			Hints: &subject.Hints{
+				TraceTime: ptypes.DurationProto(v.TraceTime),
+				API:       v.API,
+			},
+		}
+		if len(args) == 0 {
+			return log.Err(ctx, nil, "Missing argument, must specify a subject to update")
+		}
+		err := s.Search(ctx, idOrApkName(args[0]).Query(), func(ctx context.Context, entry *subject.Subject) error {
```
Comment (pmuetschard) [inline cmd/robot/subject.go]: you can combine this into:
```
if err := ...; err != nil {
  ...
}
```

File: cmd/robot/build.go (hunk 1 above)
Comment (baldwinn860) [reply cmd/robot/build.go]: ok done.

Comment (pmuetschard) [reply cmd/robot/build.go]: uhm... not done.

Comment (ben-clayton) [reply cmd/robot/build.go]: When the logic spans multiple lines, like this does - I actually encourage splitting the assignment and condition to separate statements, just for parsabilities sake.

Comment (pmuetschard) [reply cmd/robot/build.go]: OK, good to know, thanks.

Nick, it's already approved, so you can just submit as is 😄

Comment (baldwinn860) [reply cmd/robot/build.go]: ok I changed it back just to make it all match.

**Pull request merged**